driver-core/src/main/com/mongodb/client/model/changestream/ChangeStreamDocument.java

```diff
@@ -93,6 +97,15 @@ public TDocument getFullDocument() {
     }
 
     /**
+     * Returns a document containing just the _id of the changed document
```

Comment:
Can we update this / clarify for sharded collections:

```
   * For unsharded collections this contains a single field, _id, with the
   * value of the _id of the document updated.  For sharded collections,
   * this will contain all the components of the shard key in order,
   * followed by the _id if the _id isn’t part of the shard key.
```